src/main/java/com/example/bookstoreapp/filters/AuthenticationFilter.java

```diff
@@ -19,7 +14,7 @@
 @Slf4j
 public class AuthenticationFilter implements Filter {
 
-  private static final String[] PUBLIC_URI_PATTERNS = {"/api/**", "/user/login", "/user/signup", "/health"};
+    private static final String[] PUBLIC_URI_PATTERNS = {"/**", "/api/**", "/user/login", "/user/signup", "/health"};
```

Comment:
DONT apply /** .
Use login flow.

Reply:
This is for development purpose I disabled the filter so I don't have to gone through auth process it will be removed after dev process ended this page was pushed unintendedly.

Reply:
disabling auth flow will break few things. such as we return userId from userContextService.
If you disable auth flow the userId will be returned null. which may cause null pointer exception.

Reply:
Understood its value and will use the data from context during the order service implementation